src/TaskManager/Database/TaskDispatchEventRepository.cs

```diff
@@ -109,5 +109,21 @@ public async Task<bool> RemoveAsync(string taskExecutionId)
                 return false;
             }
         }
+
+        public async Task<TaskDispatchEventInfo> UpdateTaskPluginArgsAsync(TaskDispatchEventInfo taskDispatchEventInfo, Dictionary<string, string> pluginArgs)
+        {
+            Guard.Against.Null(taskDispatchEventInfo, nameof(taskDispatchEventInfo));
```

Comment:
minor thing but dont need to put nameof(taskDispatchEventInfo)) in here

Guard.Against.Null(taskDispatchEventInfo) is good enough and dont risk losing proper name of.